lib/iris/fileformats/pp_rules.py

```diff
@@ -902,6 +905,22 @@ def _all_other_rules(f):
             (AuxCoord('son', long_name='season', units='no_unit'),
              None))
 
+    # Special case where year is zero and months match.
+    # Month coordinates (--> scalar coordinates)
+    if (f.lbtim.ib == 2 and f.lbtim.ic in [1, 2, 4] and
+            ((len(f.lbcode) != 5) or
+             (len(f.lbcode) == 5 and
+              f.lbcode.ix not in [20, 21, 22, 23] and
+              f.lbcode.iy not in [20, 21, 22, 23])) and
+            f.lbyr == 0 and f.lbyrd == 0 and
+            f.lbmon == f.lbmond):
+        aux_coords_and_dims.append(
+            (AuxCoord(f.lbmon, long_name='month_number', units='no_unit'),
```

Comment:
Ok, I'll add the month abbreviation. `coord_categorisation` also has `add_month_fullname'. Is it worth adding that too?